tests/suites/test_suite_oid.function

```diff
@@ -32,3 +32,26 @@ void oid_get_certificate_policies( data_t * oid, char * result_str )
     }
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void oid_get_extended_key_usage( data_t * oid, char * result_str )
```

Comment:
style: we tend to put the `*` next to the variable, without spaces between it and the variable name.

Reply:
It did look strange to me, but I took the example from [here](https://github.com/ARMmbed/mbedtls/blob/development/tests/suites/test_suite_x509parse.function#L166), so I thought that perhaps the tests are exception